### ISSUE TITLE
Ghost entry in Scheduler dashboard

### ISSUE THREAD

**Issue by steve-dfw** [arthurdent75/SimpleScheduler#163]
Have had this ghost entry for a while. Not sure what caused it to get created. If you click on the edit button it opens a blank config dialog on the right where all other entries have the config settings for that particular schedule. There is only the <<< button to close the entry.
Have noticed that no entry below this ghost entry ever works. They have to be move above it. I tried doing the “rebuild” and uninstalled/reinstalled and the entry remains. I looked on the share and there are 5 json schedule files.

![Scheduler_Dashboard](https://github.com/user-attachments/assets/250164a0-936c-44fc-b597-bba15214689b)

**Comment by arthurdent75** [comment]
Hi @steve-dfw, allow me to say that your issue is very odd.
You correctly check the presence of another JSON, that would be the only logical explanation.
I suggest you to do this:

- backup the JSON files and the _option.dat_ from the _share/simplescheduler_ folder to your computer
- uninstall the addon
- remove the folder _simplescheduler_
- install the addon and check that the dashboard is completely blank
- the setup process creates the folder again
- Copy only the JSON files and check the dashboard. (you don't need to stop the addon, just reload the page)
- optionally copy the option.dat and restart the addon, but it would be better to set the option from the panel.

Let me know if it works.

**Comment by steve-dfw** [comment]
That fixed it!   I had already uninstalling the add-on previously, but did not remove the folder after the uninstall. 
This time after the install and folder removal, I reinstalled, started service, checked dashboard to confirm empty, copied back just the schedule item JSON files, and refreshed the page.  The 5 proper schedule items were there as they should be.  
The only thing I noticed that is different is that now in the folder there is no options.json file (only options.dat). Maybe that file was from an earlier  version.   
Thank you for the help!
